### ISSUE TITLE
Configuration examples not workng

### ISSUE THREAD

**Issue by skrobinson** [platisd/clang-tidy-pr-comments#29]
To start using `clang-tidy-pr-comments` in a project, I copied the example code in "Using this Action to safely perform analysis of pull requests from forks". I kept getting fatal git errors in `git diff -U0 HEAD^`.

I found a solution that I hope will be added to the examples to help others. `action/checkout@v2` (and `v3`) can use `fetch-depth: 2` to enable enough context for `git diff` to work.

**Comment by platisd** [comment]
Can you please create a pull request with the proposed change?
@oleg-derevenetz FYI

**Comment by oleg-derevenetz** [comment]
Yeah, in older versions of `checkout` action "no specified depth" means "depth 0" while in modern versions it means "depth 1".

**Comment by platisd** [comment]
Sounds good. Please feel free to submit a PR
